testing-guidance/workflow-tests/multiplayer/multiplayer-workflow-tests.md

```diff
@@ -19,16 +24,291 @@ These tests are designed to cover the basic functionality related to running a l
+| **Build New Project Launchers**          | <ol><li>Create a new multiplayer project.</li><li>Build all 3 launchers using CLI.<ul><li>`cmake --build build/windows --target MyProject.GameLauncher MyProject.ServerLauncher MyProject.UnifiedLauncher --config profile -- -m`</li></ul></li></ul></ol>                                                                                                  | <ul><li>Project launchers build successfully.</li><li>Server, Client and Unified launchers are built in the project build directory.</li><li>Project can be launched in **O3DE Editor** once built.</li></ul>                                                                |
+| **Build Project with Project Manager**   | <ol><li>Launch `o3de.exe`.</li><li>Add or create a project with the Multiplayer Gem enabled.</li><li>Select Build Project.</li></ol>                                                                                                                                                                                                                        | <ul><li>Project launchers build successfully.</li><li>Server, Client and Unified launchers are built in the project build directory.</li><li>Project can be launched in **O3DE Editor** once built.</li></ul>                                                                |
+| **Test Hosting with Unified Launcher**   | <ol><li>Build an existing or new multiplayer project's launchers - using either Project Manager or CLI.</li><li>Launch the `project.UnifiedLauncher`.</li><li>Open the Console `~`.</li><li>Enter `host` into the console.</li><li>Load a multiplayer level.</li><li>Launch the `project.GameLauncher`.</li><li>Enter `connect` into the console.</li></ol> | <ul><li>Console displays text confirming the session is operating in ClientServer mode.</li><li>Game Launcher client is able to connect to the ClientServer.</li><li>Project can be launched in **O3DE Editor** once built.</li></ul>                                        |
+| **Compile Autogen File**                 | <ol><li>Use either MultiplayerSample or create an autogen file following the [O3DE Tutorial: First Multiplayer Component](https://www.o3de.org/docs/learning-guide/tutorials/multiplayer/first-multiplayer-component/) instructions.</li><li>Edit the selected autogen file if using an existing one.</li><li>Build the Project.</li></ol>                  | <ul><li>No compile errors when building.</li><li>If the edits included a component field/component, the changes show in the Editor.</li><li>Project can be launched in **O3DE Editor** once built.</li></ul>                                                                 |
+---
+
+### Area: Client-Server Communications
```

Comment:
should this section have a product output and time box ?

Reply:
Nice catch. Added.